app/scenes/channel/channel_dropdown/channel_dropdown.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes} from 'react';
+import {ScrollView, Text, TouchableOpacity, View} from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+function ChannelDropdown(props) {
```

Comment:
Why aren't we doing this as a component and pass the `items` as a prop so it can be easily reused later?

Also can you please move these files under the `components` folder?

Please keep in mind that any component that can be re-used in other scenes should be included in the root `components` folder and the ones that are just related to the scene in a `components` sub-folder.